### PR TITLE
first version of task m

### PR DESCRIPTION
**Pull request by MaximOkleev**

### CODE REVIEW

File: learning_a_programming_language/loop_operator/m.c [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+
+int main(){
+	int n = 0;
+	int b = 0;
+	int c = 0;
+	scanf("%d", &n);
```
Comment (justas05) [inline learning_a_programming_language/loop_operator/m.c]: add empty line after definitions
```suggestion
	
	scanf("%d", &n);
```

File: learning_a_programming_language/loop_operator/m.c [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+
+int main(){
+	int n = 0;
+	int b = 0;
```
Comment (justas05) [inline learning_a_programming_language/loop_operator/m.c]: What does it mean b? or c?
Give the meaningful names for variables.

File: learning_a_programming_language/loop_operator/m.c [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	scanf("%d", &n);
+	scanf("%d", &b);
+	n--;
+
+	while(n > 0){
+		scanf("%d", &b);
+		if(b == 0) c++;
+		n--;
+	}
+	printf("%d",c);
```
Comment (justas05) [inline learning_a_programming_language/loop_operator/m.c]: add space after comma

File: learning_a_programming_language/loop_operator/m.c [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+	int n = 0;
+	int b = 0;
+	int c = 0;
+	scanf("%d", &n);
+	scanf("%d", &b);
+	n--;
+
+	while(n > 0){
+		scanf("%d", &b);
+		if(b == 0) c++;
```
Comment (justas05) [inline learning_a_programming_language/loop_operator/m.c]:
```suggestion
		if(b == 0)
		{
		    c++;
		}
```

**Review by justas05 — CHANGES_REQUESTED** [review]

File: learning_a_programming_language/loop_operator/m.c (hunk 3 above)
Comment (MaximOkleev) [reply learning_a_programming_language/loop_operator/m.c]: What is it comma?

File: learning_a_programming_language/loop_operator/m.c [hunk 5]
```diff
@@ -2,17 +2,22 @@
 
 int main(){
 	int n = 0;
-	int b = 0;
-	int c = 0;
+	int schit = 0;
```
Comment (justas05) [inline learning_a_programming_language/loop_operator/m.c]: schit? What does it mean?

File: learning_a_programming_language/loop_operator/m.c [hunk 6]
```diff
@@ -2,17 +2,22 @@
...
 	scanf("%d", &n);
-	scanf("%d", &b);
+	scanf("%d", &schit);
 	n--;
 
 	while(n > 0){
-		scanf("%d", &b);
-		if(b == 0) c++;
+		scanf("%d", &schit);
+		if(schit == 0)
```
Comment (justas05) [inline learning_a_programming_language/loop_operator/m.c]: Space after if

File: learning_a_programming_language/loop_operator/m.c [hunk 7]
```diff
@@ -2,17 +2,22 @@
...
+		{
+		    count_zero++;
+		}
 		n--;
 	}
-	printf("%d",c);
+	
+	printf("%d",count_zero);
```
Comment (justas05) [inline learning_a_programming_language/loop_operator/m.c]: Space after comma

**Review by justas05 — APPROVED** [review]

**Pull request merged**